frontend/src/components/Header.vue

```diff
@@ -33,58 +33,56 @@
                     </v-list>
                 </v-col>
             </v-row>
-          </v-menu>
+          </v-menu> -->
+
+          <v-btn v-if="!loggedIn" depressed text large id="login-btn" class="hidden-sm-and-down" :href="logInUrl" @click="clearStorage" height="100%">{{$tc("LogIn")}}</v-btn>
```

Comment:
This should probably be a modification to the User component instead

Reply:
Was a pain to adjust the styling when it was in there, I don't feel like it's worth dealing with on our current timeline

Reply:
It's incurring technical debt to do so as it's more to manage in the 4 versions in the header

Reply:
Agreed but not worth time investment for mvp